embedded/servo_bq/servo_balsaq.cc

```diff
@@ -0,0 +1,50 @@
+//%bin(servo_balsaq_main)
+#include "embedded/servo_bq/servo_balsaq.hh"
+#include "infrastructure/balsa_queue/bq_main_macro.hh"
+
+#include <chrono>
+#include <cstddef>
+#include <iostream>
+#include <sstream>
+#include <cstdlib>
+
+
+//%deps(balsa_queue)
+//%deps(message)
+
+namespace jet {
+
+ServoBq::ServoBq() {
+  set_comms_factory(std::make_unique<MqttCommsFactory>());
+}
+constexpr int PWM_FREQUENCY = 330;
+
+void ServoBq::init(int argc, char *argv[]) {
+  servo_index = atoi(argv[1]); // get the servo number in 0...3 from the command line
```

Comment:
I know I said "whatever don't worry about it" but you should really do this just in case: https://stackoverflow.com/a/3024202